Example/IntegrationTests/Push/PushTests.swift

```diff
@@ -167,6 +167,7 @@ final class PushTests: XCTestCase {
                 return
             }
             subscriptionTopic = pushSubscription.topic
+            sleep(1)
```

Comment:
nit: can you leave context here of why this is required and what is being fixed?